src/military/military.cpp

```diff
@@ -5161,6 +5193,9 @@ void update_land_battles(sys::state& state) {
 		auto defender_per = state.world.leader_get_personality(state.world.land_battle_get_general_from_defending_general(b));
 		auto defender_bg = state.world.leader_get_background(state.world.land_battle_get_general_from_defending_general(b));
 
+		auto atk_leader_exp_mod = (1 + attacker_per.get_experience()) * (1 + attacker_bg.get_experience());
```

Comment:
you should sum the personality and background experience values: the game always takes the leaders stats to be the sum of the personality and background

Reply:
Fixed w #13ae911